src/unix.rs

```diff
@@ -32,6 +32,9 @@ use tokio_io::IoFuture;
 
 pub use self::libc::{SIGUSR1, SIGUSR2, SIGINT, SIGTERM};
 pub use self::libc::{SIGALRM, SIGHUP, SIGPIPE, SIGQUIT, SIGTRAP};
+#[cfg(any(target_os = "dragonfly", target_os = "freebsd", target_os = "macos",
+          target_os = "netbsd", target_os = "openbsd"))]
+pub use self::libc::SIGINFO;
```

Comment:
Hey @asomers, thanks for the PR!

I'm a little hesitant for just adding this directly at the root of the crate because it can cause some cross platform confusion (i.e. importing this on a BSD-like system will fail to compile on linux for example).

I think we can probably conditionally expose a `os::bsd` module, document it as OS-specific, and export the signal there.

Reply:
Ah yes, I was thinking at first the module would be exposed from the root, but this is fine too!